transcom/mymove
# B-21426: Add is_oconus bool column to address INT

## [B-21426 Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21426)

## Summary

Adding is_conus boolean column to the DB. Leaving as nullable for now, to be made not nullable after [21427](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21427), which has been added to the AC for that BL.

### How to test

1. Pull the branch, run db_dev_migrate, verify that you see the new is_oconus column on the addresses table.

![image](https://github.com/user-attachments/assets/ee59fcf9-88ad-4c4b-b121-8379f0228050)
